pkg/asset/installconfig/gcp/validation.go

```diff
@@ -0,0 +1,85 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	compute "google.golang.org/api/compute/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/openshift/installer/pkg/types"
+)
+
+// Validator provides validation which requires access to the GCP API.
+type Validator struct {
+	Client        API
+	InstallConfig *types.InstallConfig
+}
+
+// Validate executes platform-specific validation.
+func (v *Validator) Validate() field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	allErrs = append(allErrs, v.validateVPC()...)
```

Comment:
looking at https://github.com/openshift/installer/blob/master/pkg/types/validation/installconfig.go#L78

the call site passes the field to the function for that section.
So best would be `allErrs = append(allErrs, validatePlatform(config.Platform.GCP, field.NewPath("platform").Child("gcp"), client)`

Reply:
Ok I will fix this throughout